src/Scene/NodeProcess.js

```diff
@@ -217,6 +226,39 @@ function findAncestorWithValidTextureForLayer(node, layerType, layer) {
     }
 }
 
+const nullMesh = new THREE.Mesh();
+
+function updateNodeFeature(quadtree, node, layersConfig) {
+    const featureLayers = layersConfig.getGeometryLayers();
+    const promises = [];
+    for (let i = 0; i < featureLayers.length; i++) {
+        const layer = featureLayers[i];
+        const protocol = layer.protocol;
+        if (protocol.toLowerCase() == 'wfs') {
+            if (layer.tileInsideLimit(node, layer) && !node.content.get(layer.id)) {
+                const args = {
+                    layer,
+                };
+                const promise = quadtree.interCommand.request(args, node, refinementCommandCancellationFn).then((result) => {
+                    // if request return empty json, WFS_Provider.getFeatures return undefined
+                    if (result.feature) {
+                        result.feature.layer.root.add(result.feature);
+                        node.content.set(layer.id, result.feature);
+                    }
+                    else {
+                        node.content.set(layer.id, nullMesh);
+                    }
+                })
+                .catch(() => {
+                // Command has been canceled, no big deal, we just need to catch it
```

Comment:
Nope. You can't just silently catch **all** errors. It looks like the good old Java pattern :-) 
``` java
try {
   // ...
} catch (Exception ex) {
  // something went wrong, oops
}
```
 
Hopefully PR #202  will help.